test/testContainers/localstack.js

```diff
@@ -9,16 +9,19 @@ export default class Localstack {
 
   async start () {
     console.log('Starting LocalstackContainer')
-    this.container = await new LocalstackContainer(this.image).start()
+    this.container = await new LocalstackContainer(this.image).withReuse(true).start()
```

Comment:
need to add withReuse here as it stops testcontainers managing the closing of the docker container. this way we can manually get the same container later and call stop. otherwise, this happens too early causing tests to fail

Reply:
Thanks for explaining.  I think the `withReuse` just means it will re-use the container created previously without stopping, which ends up being a small performance optimisation.  If the container was stopping before your test run then perhaps the scoping isn't right or there's a timeout you need to increase / remove.  Anyhow, there should be no harm using the same localstack container since there shouldn't be any residual state issues.